bike-erp/src/components/PermissionManagement/UserAccountsTable.tsx

```diff
@@ -1,18 +1,24 @@
-import { Button, Table, TableBody, TableRow } from "@material-ui/core";
-import SaveIcon from "@material-ui/icons/Save";
 import axios from "axios";
 import { useEffect, useState } from "react";
+
```

Comment:
Remove space here too

Reply:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍